policyengine_us/parameters/gov/states/mt/tax/income/exemptions/applies.yaml

```diff
@@ -0,0 +1,14 @@
+description: Montana applies a state-specific exemption structure if this is true.
+values:
+  2021-01-01: true
+  2024-01-01: false
+metadata:
+  reference:
+    - title: "Montana Tax Reform Update (2024)"
```

Comment:
please add tax form (might be even better than the instructions) for 2023 and 2024 if applicable - to show the diff